### PR TITLE
Contracts page: Add blurb

### LINKED ISSUE #376: Add new blurb to /contracts page about GTFS compliance
New design here: https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=3979%3A11994

Copy here: We recommend creating an action plan for becoming GTFS-ready before going contactless, as the vendors you select can help you use this data for reporting purposes plus help your customers see your real-time vehicle location and fare information on journey-planning smartphone maps and apps.

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #376 

## What this PR does
- Typography: Adds `.h4` class
- Typography change: Make the css better reflect the new Type guide on Figma. The font family for H1 - H3 is Raleway, and H4 and below is Poppins.
- Typography: Adds `.subtext` class to represent the `Highly recommended` type style, to be used with `<em>` or `<span>`

## Screenshots
image
image

### CODE REVIEW

File: src/_sass/_base.scss [hunk 1]
```diff
@@ -58,10 +71,17 @@ h3 {
   font-size: calc(24rem / 16);
 }
 
-h4 {
+h4,
+.h4 {
   font-size: calc(16rem / 16);
 }
 
+.italics {
```
Comment (machikoyasuda) [inline src/_sass/_base.scss:79]: @segacy1 I've added a style called `italics` that represents the italicized, `#136c97`, 14px size font. This is not on the Figma Style Guide/Type section. https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=3725%3A10998&t=dlLp11tKHkYYCsyF-0

File: src/contracts/index.html [hunk 2]
```diff
@@ -45,14 +45,15 @@
...
-  The State of California has Master Service Agreements (MSAs) available to help you modernize your transit system.
-  We recommend creating an action plan for becoming GTFS-ready before going contactless, as the vendors you select
-  can help you use this data for reporting purposes plus help your customers see your real-time vehicle location and
-  fare information on journey-planning smartphone maps and apps.
+<p class="pb-3">
+  The State of California has Master Service Agreements (MSAs) available to help you modernize your transit system.  To add contactless fare collection, you will need up to three new technologies and ensure internet connectivity on your fleet (if you don’t already have it).
 </p>
 
-<p class="my-4">
+<strong class="h4">Become GTFS Compliant first for best results</strong> <em class="italics">Highly recommended</em>
```
Comment (machikoyasuda) [inline src/contracts/index.html:52]: Just wanted to note here @mrose914 @esquared415 that `GTFS Compliant` and `GTFS-compliant` are formatted in two different ways on this copy as it is now:

image

**Comment by machikoyasuda** [comment]
This PR is ready for design review @segacy1

File: src/_sass/_base.scss (hunk 1 above)
Comment (segacy1) [reply src/_sass/_base.scss:79]: Hi @machikoyasuda the blue color was an error! We can just make that our primary blue. I'll update in figma. 

For the italics - it's the "subtext" style italicized, so I considered it more of an override than its own style. I think calling italics works atm!

Comment (machikoyasuda) [reply src/_sass/_base.scss:79]: Glad I asked :) Thank you for clarifying and responding quickly!

**Comment by machikoyasuda** [comment]
This PR is now ready for code review @thekaveman

**Comment by segacy1** [comment]
@machikoyasuda design looks good to me! 

I think we change the new H4 text copy to "GTFS-compliant" since that's what is currently on our site. I've updated in Figma.
cc: @mrose914

**Comment by machikoyasuda** [comment]
@segacy1 Spelling fixed thank you!

**Review by segacy1 — APPROVED** [review]

**Review by thekaveman — APPROVED** [review]

**Pull request merged**